delivery_carrier_label_postlogistics/wizards/postlogistics_oauth.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+import json
+import requests
+
+from odoo import models, fields, api
+
+
+class PostlogisticsAuth(models.TransientModel):
+    _name = 'postlogistics.auth'
+    _description = 'Postlogistics Auth'
+
+    client_id = fields.Char(
```

Comment:
Yes, I defined directly in view. Even if the reset function don't works.

Reply:
Why define the required in the view ? For me it's better in the model no ? :thinking:

Reply:
Just say before, if I define the required in the model (instead of in the view), the reset function don't works, because the function empty this field, and this field is required.
I will add a comment into the code.